app/routes/sessions/index.ts

```diff
@@ -2,17 +2,21 @@ import Store from "@ember-data/store";
 import Route from "@ember/routing/route";
 import { service } from "@ember/service";
 import { tracked } from "@glimmer/tracking";
+import MunicipalityListService from "frontend-burgernabije-besluitendatabank/services/municipality-list";
+import { seperator } from "frontend-burgernabije-besluitendatabank/components/filter-sidebar/select-multiple-filter";
```

Comment:
**Issue:** A route shouldn't be dependents of a component. What about moving `seperator` in a constants file and use it in both `select-multiple-filter` component and `session.index` route?

Reply:
Good idea! Created `helpers/constants.ts`. Don't know if that's the optimal location though, so feel free to move it if desired